index.js

```diff
@@ -0,0 +1,157 @@
+const show = document.querySelector(".show"),
+      startButton = document.querySelector(".start-btn"),
+      noShow = document.querySelector(".no-show"),
+      retryButton = document.querySelector(".retry-btn"),
+      writeNum = document.querySelector(".write-num"),
+      resultList = document.querySelector(".result-list"),
+      message = document.querySelector(".message");
+
+let tryNum = 0;
+
+function genNum() {
+   (function() {
+      show.classList.add("no-show");
+      noShow.classList.remove("no-show");
+   })();
+
+   let randomNum = `${Math.floor(Math.random() * 1000)}`;
+
+   if(randomNum.length === 1) {
+      genNum();
+   } else if(randomNum.length === 2) {
+      randomNum = 0 + randomNum;
+   } else {
+      randomNum = randomNum;
+   }
+
+   const a = randomNum[0];
+   const b = randomNum[1];
+   const c = randomNum[2];
+
+   function saveNum(num) {
+      localStorage.setItem("Random Number", num);
+   }
+
+   if((a !== b) && (b !== c) && (c !== a)) {
```

Comment:
Strict equal을 사용하셨군요👍  아래에 Strict equal과 관련한 링크를 참조하시면 좋을 것 같아요.
단, a, b ,c 등의 변수명은 보다 명확하면 좋을 것 같아요.

```suggestion
   if(firstNum !== secondNum) && (secondNum !== thirdNum) && (thirdNum !== firstNum)) {
```

- [MDN: Strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)
- [MDN: Equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)
- [javascript-double-equals-vs-triple-equals](https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a)